docs/team/longngng.md

```diff
@@ -2,5 +2,48 @@
 
 ## Overview
 
+**Zer0Note** is a note taking and organisation application meant for people who love typing. (especially those who can type fast).
+It combines the features of note-taking apps (like OneNote) with the interaction speed of command-line based tools like vim and emacs.
+It helps users to keep track of deadlines and take notes quickly.
 
-### Summary of Contributions
+## Summary of Contributions
+
+### Code contributed
+
+See the [RepoSense Dashboard here](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=longngng&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other) for my contributions to Zer0Note. 
+
+### Enhancements implemented
+
+* Implemented the `list` command for timetable mode in v1.0 
+* Implemented `mode` command in v1.0
+* Implemented the `list` command for notebook mode in v2.0 
+* Implemented `list /urgent` command in v2.0 to allow users view their top three upcoming deadlines.
+
+### Contributions to User Guide
+
+* Wrote the section for `list` command in `Notebook` mode. 
+* Wrote the section for `list` command in `Timetable` mode
+* Wrote the section for `mode` command
+* Wrote the section for `exit` command
+
+### Contributions to Developer Guide
+
+* Added details and sequence diagrams for `list` command in `Notebook` mode
+* Added details and sequence diagrams for `list` command in `Timetable` mode
+* Added details and sequence diagrams for `ModeSwitch` feature
+* Added details for `Task` components and `Notebook` components
+
+### Contributions to team-based tasks
+
+* I created the first release of the jar file.
+* I refactored the code once at the beginning and introduce the AppState object to improve the use of OOP principles.
+* I created a demo video for the first release to present during the tutorial
```

Comment:
nice haha

Reply:
LOL

Reply:
Maybe you can link like the Pull requests?